x/qgb/README.md

```diff
@@ -118,6 +118,42 @@ if k.CheckLatestAttestationNonce(ctx) && k.GetLatestAttestationNonce(ctx)+1 != n
 }
 ```
 
+## Verification command
+
+The QGB verification command is part of the `celestia-appd` binary. It allows the user to verify that a set of shares has been posted to a specific QGB contract.
+
+```shell
+$ celestia-appd verify --help          
+                              
+Verifies that a transaction hash, a range of shares, or a blob referenced by its transaction hash were committed to by the QGB contract
+
+Usage:
+  celestia-appd verify [command]
+
+Available Commands:
+  blob        Verifies that a blob, referenced by its transaction hash, in hex format, has been committed to by the QGB contract. Only supports one blob for now
+  shares      Verifies that a range of shares has been committed to by the QGB contract
+  tx          Verifies that a transaction hash, in hex format, has been committed to by the QGB contract
+
+Flags:
+  -h, --help   help for verify
+
+Global Flags:
+      --home string         directory for config and data (default "/home/midnight/.celestia-app")
+      --log_format string   The logging format (json|plain) (default "plain")
+      --log_level string    The logging level (trace|debug|info|warn|error|fatal|panic) (default "info")
+      --trace               print out full stack trace on errors
+
+Use "celestia-appd verify [command] --help" for more information about a command.
+
+```
+
+It currently supports three sub-commands:
+
+- `blob`: Takes a transaction hash, in hex format, and verifies that the blob paid for by the transaction has been committed to by the QGB contract. It only supports one blob for now.
```

Comment:
> It only supports one blob for now.

does this mean that it only supports proving to one blob at a time, or only being supports PFBs that pay for a single blob?

Reply:
It means that if a PFB has one blob, it will prove it. If it has multiple blobs, then it will prove the first one.
Extending it to prove multiple blobs should be no biggy, I guess there is already an issue for that.

Reply:
> might also want to link to nmt/rsmt2d/specs, and eventually a diagram would likely be useful to show precising what inclusion proofs are being used for which proof since we're using two or three different proofs

https://github.com/celestiaorg/celestia-app/issues/1313